src/redisearch_rs/ffi/.gitignore

```diff
@@ -0,0 +1,3 @@
+data/*
+target/
+flamegraph.svg
```

Comment:
?

Reply:
c&p mistake. I created the `ffi` crate by copying an existing one.